blocks/algorithms/__init__.py

```diff
@@ -169,6 +170,10 @@ def __init__(self, step_rule=None, gradients=None, **kwargs):
                 zip(self.params, tensor.grad(self.cost, self.params))))
         self.step_rule = step_rule if step_rule else SteepestDescent()
 
+        self.total_gradient_norm = named_copy(
```

Comment:
Cool, can you make a ticket for it? (I'm on my phone)
On Jan 22, 2015 9:04 AM, "Dmitry Bogdanov" <notifications@github.com> wrote:

> In blocks/algorithms/__init__.py
> <https://github.com/bartvm/blocks/pull/131#discussion_r23376582>:
>
> > @@ -169,6 +170,10 @@ def __init__(self, step_rule=None, gradients=None, **kwargs):
> >                  zip(self.params, tensor.grad(self.cost, self.params))))
> >          self.step_rule = step_rule if step_rule else SteepestDescent()
> >
> > +        self.total_gradient_norm = named_copy(
>
> Yep, I also had it mind. I see no problem in adding it a bit later, and
> seems we can release 0.01 without it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bartvm/blocks/pull/131/files#r23376582>.
>